src/Outsystems.SetupTools/Functions/Set-OSServerPerformanceTunning.ps1

```diff
@@ -52,165 +138,215 @@ function Set-OSServerPerformanceTunning
             return
         }
 
-        if ($(-not $osVersion) -or $(-not $(GetServerInstallDir)))
+        $SkipPlatformCheck = $AdvancedConfigurations.SkipPlatformCheck
+        $SkipProcessSchedulingConfig = $($AdvancedConfigurations.Sections.ProcessSchedulingConfig.ShouldBeSkipped)
+        $SkipNETConfig = $($AdvancedConfigurations.Sections.NETConfig.ShouldBeSkipped)
+        $SkipIISUploadSizeLimitsConfig = $($AdvancedConfigurations.Sections.IISUploadSizeLimitsConfig.ShouldBeSkipped)
+        $SkipIISConnectionsConfig = $($AdvancedConfigurations.Sections.IISConnectionsConfig.ShouldBeSkipped)
+        $SkipAppPoolsConfig = $($AdvancedConfigurations.Sections.AppPoolsConfig.ShouldBeSkipped)
+        $SkipMoveAppsToOSAppPools = $($AdvancedConfigurations.Sections.AppPoolsConfig.SkipMoveAppsToOSAppPools)
+
+        if ($(-not $SkipPlatformCheck) -and ($(-not $osVersion) -or $(-not $(GetServerInstallDir))))
         {
             LogMessage -Function $($MyInvocation.Mycommand) -Phase 1 -Stream 3 -Message "Outsystems platform is not installed"
             WriteNonTerminalError -Message "Outsystems platform is not installed"
 
             return
         }
 
-        # Configure process scheduling -- http://technet.microsoft.com/library/Cc976120
-        LogMessage -Function $($MyInvocation.Mycommand) -Phase 1 -Stream 0 -Message "Configuring Windows processor scheduling priority to background services"
-        try
-        {
-            RegWrite -Path "HKLM:\HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\PriorityControl" -Name "Win32PrioritySeparation" -Type "Dword" -Value 24
-        }
-        catch
+        if (-not $SkipProcessSchedulingConfig)
         {
-            LogMessage -Function $($MyInvocation.Mycommand) -Phase 1 -Exception $_.Exception -Stream 3 -Message "Error configuring processor scheduling priority"
-            WriteNonTerminalError -Message "Error configuring processor scheduling priority"
+            # Configure process scheduling -- http://technet.microsoft.com/library/Cc976120
+            LogMessage -Function $($MyInvocation.Mycommand) -Phase 1 -Stream 0 -Message "Configuring Windows processor scheduling priority to background services"
+            try
+            {
+                RegWrite -Path "HKLM:\HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\PriorityControl" -Name "Win32PrioritySeparation" -Type "Dword" -Value 24
```

Comment:
We should move this to a constant in src/Outsystems.SetupTools/Lib/Constants.ps1

Note: This maps to the registry path/value in src/Outsystems.SetupTools/Functions/Get-OSPerfTuningStatus.ps1